include/bluetooth/services/latency.rst

```diff
@@ -0,0 +1,38 @@
+.. _latency_readme:
+
+GATT Latency Service
+####################
+
+The GATT Latency Service is a custom service that receives data and returns result about GATT latency operations.
```

Comment:
Would rephrase it to:
"The GATT Latency Service is a custom service containing a single writable characteristic. This characteristic may be used to calculate the latency of a GATT Write operation"